### DISCUSSION TITLE
Push translate works to crowdin

### DISCUSSION THREAD

**Discussion by bluehomewu** [mackie100/OCClocalizations#214] [General]
Can we push Translate works to crowdin,make more people help us?
I have students package for crowdin.
If you need, please tell me.😊
And also I'm working for Chinese Traditional right now,Thanks for working.

**Comment by btwise** [comment]
@bluehomewu 可以，等你繁体部分全部完成！

**Comment by btwise** [comment]
@mackie100 
Please add  Chinese Traditional  localization framework!

**Comment by bluehomewu** [comment]
> @bluehomewu 可以，等你繁体部分全部完成！

Thanks.
我剩下最後一個部分 (剩餘7個檔案)

**Comment by mackie100** [comment]
> Can we push Translate works to crowdin,make more people help us?
> I have students package for crowdin.
> If you need, please tell me.😊
> And also I'm working for Chinese Traditional right now,Thanks for working.

Welcome @bluehomewu, I added Chinese Traditional localization folder from Chinese Simplified folder. Feel free to update any time when you ready. Thanks for your work.

**Comment by bluehomewu** [comment]
> > Can we push Translate works to crowdin,make more people help us?
> > I have students package for crowdin.
> > If you need, please tell me.😊
> > And also I'm working for Chinese Traditional right now,Thanks for working.
> 
> Welcome @bluehomewu, I added Chinese Traditional localization folder from Chinese Simplified folder. Feel free to update any time when you ready. Thanks for your work.

Well,can you wait me make PR,I will make great PR let you just merge,you can see it in mine repo,there's a little bit different now,so....I hope you can revert commit back.
If you keep this commit,right now,maybe when you merge you will get conflict.
Have a nice day

**Comment by mackie100** [comment]
> > > Can we push Translate works to crowdin,make more people help us?
> > > I have students package for crowdin.
> > > If you need, please tell me.😊
> > > And also I'm working for Chinese Traditional right now,Thanks for working.
> > 
> > 
> > Welcome @bluehomewu, I added Chinese Traditional localization folder from Chinese Simplified folder. Feel free to update any time when you ready. Thanks for your work.
> 
> Well,can you wait me make PR,I will make great PR let you just merge,you can see it in mine repo,there's a little bit different now,so....I hope you can revert commit back.
> If you keep this commit,right now,maybe when you merge you will get conflict.
> Have a nice day

How can I resolve? Removing last commit?

**Comment by bluehomewu** [comment]
> > > > Can we push Translate works to crowdin,make more people help us?
> > > > I have students package for crowdin.
> > > > If you need, please tell me.😊
> > > > And also I'm working for Chinese Traditional right now,Thanks for working.
> > > 
> > > 
> > > Welcome @bluehomewu, I added Chinese Traditional localization folder from Chinese Simplified folder. Feel free to update any time when you ready. Thanks for your work.
> > 
> > 
> > Well,can you wait me make PR,I will make great PR let you just merge,you can see it in mine repo,there's a little bit different now,so....I hope you can revert commit back.
> > If you keep this commit,right now,maybe when you merge you will get conflict.
> > Have a nice day
> 
> How can I resolve? Removing last commit?

Revert or Delete commit I think it's ok.

**Comment by btwise** [comment]
@mackie100 
OpenCorePkg Added `ForceBooterSignature` quirk for Mac EFI

**Comment by btwise** [comment]
@bluehomewu 
You submit it to me in traditional Chinese,https://github.com/btwise/OCClocalizations  and I will review it and push it to @mackie100

**Comment by bluehomewu** [comment]
> @bluehomewu
> You submit it to me in traditional Chinese,https://github.com/btwise/OCClocalizations and I will review it and push it to @mackie100

Sorry to disturb you,but why I can't direct push to @mackie100 
My repo is fork from @mackie100 so make PR to him will be more easy.

**Comment by btwise** [comment]
> > @bluehomewu
> > 您将其以繁体中文提交给我，[https：//github.com/btwise/OCClocalizations](https://github.com/btwise/OCClocalizations)，我将对其进行审核并将其推送至@ mackie100
> 
> 抱歉打扰您，但是为什么我不能直接推送到@ mackie100
> 我的仓库是@ mackie100的叉子，所以给他做PR会更容易。



> > @bluehomewu
> > You submit it to me in traditional Chinese,https://github.com/btwise/OCClocalizations and I will review it and push it to @mackie100
> 
> Sorry to disturb you,but why I can't direct push to @mackie100
> My repo is fork from @mackie100 so make PR to him will be more easy.

也可以，但他并不懂中文，如果有一些个性化翻译错误，我可以方便的修正，随你意愿了

**Comment by bluehomewu** [comment]
> > > @bluehomewu
> > > 您将其以繁体中文提交给我，[https：//github.com/btwise/OCClocalizations](https://github.com/btwise/OCClocalizations)，我将对其进行审核并将其推送至@ mackie100
> > 
> > 抱歉打扰您，但是为什么我不能直接推送到@ mackie100
> > 我的仓库是@ mackie100的叉子，所以给他做PR会更容易。
> 
> 
> 
> > > @bluehomewu
> > > You submit it to me in traditional Chinese,https://github.com/btwise/OCClocalizations and I will review it and push it to @mackie100
> > 
> > Sorry to disturb you,but why I can't direct push to @mackie100
> > My repo is fork from @mackie100 so make PR to him will be more easy.
> 
> 也可以，但他并不懂中文，如果有一些个性化翻译错误，我可以方便的修正，随你意愿了

我這邊有找了一些朋友協助我一起校正翻譯
所以應該是沒什麼問題
我自己也做過許多程式的繁體中文翻譯
所以在翻譯方面應該是不用擔心
（除了我沒注意到的部分）😂😂

**Comment by bluehomewu** [comment]
@mackie100 After merge I hope we can push strings to [Crowdin](http://crowdin.com/).
And also I still have many works,so I can't quick push new strings when you update. (Will be late,lol.)
If you push translate works to crowdin,we can quickly check everythings you update,and use our phone to start works.
That will be very nice.
Hope you have a nice day.

**Comment by bluehomewu** [comment]
I make a discussion,we can talk about at here😊
@mackie100 @btwise

**Comment by bluehomewu** [comment]
@mackie100 https://support.crowdin.com/uploading-files/
Check this website.😊

**Comment by bluehomewu** [comment]
I already check [commit](https://github.com/mackie100/OCClocalizations/commit/701475123f8f7eca125a4894da666c0627e6ad14).
I will bring up tomorrow.

**Reply by mackie100** [reply]
@bluehomewu @btwise  Getting error if use GITEE, OS-X and CSDN network, probably not synced yet.

**Reply by btwise** [reply]
What mean?

**Reply by btwise** [reply]
There is no need to use Crowdin, just use a Git repository！

**Reply by bluehomewu** [reply]
> There is no need to use Crowdin, just use a Git repository！

I hope we can push repository to Crowdin it will be more portable,when we outdoor, can use mobile phone start work 😊

**Reply by mackie100** [reply]
> What mean?

When you use traditional chinese language

**Reply by btwise** [reply]
@mackie100 
I will sync all git resources!

**Comment by bluehomewu** [comment]
@mackie100 Done.
These few days I had a lot of works,sorry to delay push translate work.
Maybe next I will do like this time,translate few commit at the same times.
Not follow yours,untill July 2021.

**Comment by bluehomewu** [comment]
@mackie100 Sorry , because some reasons I need to drop translate works.
If I can be back, I will.

**Reply by mackie100** [reply]
That's ok, no problem.

**Comment by comradekingu** [comment]
If you want contributions, use Weblate instead of Crowdin.
Happy to help if this gets a libre license.

**Comment by comradekingu** [comment]
@bluehomewu Consider Weblate as a tool and then what projects with student packages are.
